packages/react-liveness/src/components/FaceLivenessDetector/shared/MatchIndicator.tsx

```diff
@@ -38,3 +38,5 @@ export const MatchIndicator: React.FC<MatchIndicatorProps> = ({
     </div>
   );
 };
+
+export const MemoizedMatchIndicator = React.memo(MatchIndicator);
```

Comment:
Nice! Should we do the same for the feedback component?

Reply:
This was just focused on the face match percentage for now, will address that separately